test/integration/targets/old_style_modules_posix/tasks/main.yml

```diff
@@ -26,3 +26,14 @@
 - assert:
     that:
       - 'goodbye_ansible.msg == "Goodbye, Ansible!"'
+
+- name: Execute module directly
+  command: '/bin/sh {{ role_path }}/library/helloworld.sh'
+  register: direct
+  ignore_errors: true
+
+- assert:
+    that:
+      - direct is failed
+      - |
```

Comment:
Why the multi-line syntax here?

Reply:
I had done it originally due to simplifying quoting, but after a change it's no longer needed, I just didn't remove it.  It wasn't done for multiline, but rather as a block literal, to avoid the need for outer quotes which would have complicated things.